modules/react/tooltip/lib/useTooltip.tsx

```diff
@@ -145,10 +145,11 @@ export function useTooltip<T extends Element = Element>({
     onBlur: onHide,
   };
 
+  // deprecate: I don't want to remove the aria-description string
   // remove `aria-describedby` if undefined to not override what's provided
-  if (targetProps['aria-describedby'] === undefined) {
-    delete targetProps['aria-describedby'];
-  }
+  // if (targetProps['aria-describedby'] === undefined) {
+  //   delete targetProps['aria-describedby'];
+  // }
```

Comment:
If we always want `aria-description` you can remove these lines of code.

Reply:
Sounds good, I got rid of it.